src/layout/KeyValue.tsx

```diff
@@ -9,33 +9,53 @@ export interface Props { state: LiveSplit.KeyValueComponentStateJson }
 
 export default class KeyValue extends React.Component<Props> {
     public render() {
+        const keyCell = <td
+            className="key-value-key"
+            style={{
+                color: map(this.props.state.key_color, colorToCss),
```

Comment:
This makes me sad. They added ?. in TypeScript (and JavaScript), which is almost like ? in Rust, except it's completely useless outside of method calls. So this could look like this if `.toCss()` was a method, but it isn't:
```typescript
this.props.state.key_color?.toCss()
```

:(

Reply:
How would you do it in Rust?

Reply:
@wooferzfg would be pretty similar to what CryZe described

```rust
//T and U are different types
fn func(&self: T) -> Option<U> {
    self.field?.method()
}
```